### PR TITLE
Version 7

### PR DESCRIPTION
**Pull request by raitisbe**
## Description

Preparation of version 7

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [x] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

File: projects/hslayers/README.md [hunk 1]
```diff
@@ -36,7 +37,7 @@ It will install hslayers-ng for the default application specified in your angula
...
+`npm i bootstrap@^5.0.0 ol@^6.0.0 @angular/cdk@^12 @angular/common@^12 @angular/core@^12 @angular/forms@^12.0.0 @ngx-translate/core@^13 @ngx-translate/http-loader@^6.0.0 deepmerge@^4.0.0 dayjs@^1.0.0 @ng-bootstrap/ng-bootstrap@^10.0.0 ol-popup@^4.0.0 proj4@^2.6.0 share-api-polyfill@^1.0.0 @angular/compiler@^12.0.0 @angular/platform-browser@^12.0.0 @angular/platform-browser-dynamic@^12.0.0 @angular/localize@^12.0.0 rxjs@^6.0.0 zone.js@^0.11.3 xml-js@^1.0.0 ngx-cookie-service@^12.0.0 geostyler-style@^5 geostyler-sld-parser@^3 geostyler-openlayers-parser@^3 geostyler-legend@>=3 ngx-color@^7 queue resumablejs d3 geostyler-qgis-parser@^1 jszip @types/jszip polygon-splitter polygon-clipping geotiff`
```
Comment (DailisLangovskis) [inline projects/hslayers/README.md:40]: @raitisbe Geotiff is no longer need for HsLayers

Comment (raitisbe) [reply projects/hslayers/README.md:40]: How about ol ?

Comment (DailisLangovskis) [reply projects/hslayers/README.md:40]: OL requires geotiff, but it is a part of its deps, so we don't need to add it. Ran npm ci, and started test app with no problems.

P.s. It was removed at the same time when jszip was added.

**Comment by raitisbe** [comment]
Thanks, will remove geotiff then

**Pull request merged**